scripts/globals/mobskills/batterhorn.lua

```diff
@@ -0,0 +1,30 @@
+-----------------------------------
+--  Batterhorn
+--  Description: Inflicts damage in a frontal area of effect. Additional effect: Knockback.
+--  Type: Physical
+--  Utsusemi/Blink absorb: 2-3 shadows
+--  Range: Melee
+--  TODO: Verify damage type
+--  https://www.bg-wiki.com/ffxi/Locus_Wivre
+-----------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/mobskills")
+-----------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskill_object.onMobWeaponSkill = function(target, mob, skill)
+    local numhits = math.random(2, 3)
```

Comment:
"absorb 2-3 shadows" doesn't necessarily mean it's 2-3 hits, because its a conal ability it will naturally wipe 1+ shadows depending on ninjutsu skill. This is probably just one hit. Need retail verification. For now, 1 would suffice, leave in a TODO for it.